views/components/info/control.es

```diff
@@ -36,6 +38,7 @@ config.on('config.set', (path, value) => {
 
 const PoiControl = connect((state, props) => ({
   muted: get(state, 'config.poi.content.muted', false),
+  tbtriggered: get(state, 'config.poi.touchbar.triggered', null),
```

Comment:
maybe we could find better way to propagate the touchbar status instead of using config, maybe in ui store?